MoltenVK/MoltenVK/GPUObjects/MVKPipeline.mm

```diff
@@ -1533,13 +1533,14 @@ static MTLVertexFormat mvkAdjustFormatVectorToSize(MTLVertexFormat format, uint3
 	// See if a translated binding already exists (for example if more than one VA needs the same translation).
 	for (auto& xltdBind : _translatedVertexBindings) {
 		if (xltdBind.binding == binding && xltdBind.translationOffset == translationOffset) {
+			xltdBind.mappedAttributeCount += 1u;
```

Comment:
The style pattern in MoltenVK would use `++` here.

Is there an argument that the `+= 1` idiom might be better?

Reply:
Changed!

There's no reason other than personal preference, and being unsure of the code-style when I submitted the PR.